externs/shaka/player.js

```diff
@@ -1216,6 +1218,12 @@ shaka.extern.LanguageRole;
  *   width: number
  * }}
  *
+ * @property {number} imageHeight
+ *    The image height in px. The image height could be different to height if
+ *    the layout is different to 1x1.
+ * @property {number} imageWidth
+ *    The image width in px. The image width could be different to height if
```

Comment:
Nitpick: This should say "the image width could be different to width" rather than "the image width could be different to height"

Reply:
Done!